version1/02 - Object types.md

```diff
@@ -609,6 +615,102 @@ The following notation is used in the tables below:
+|           |           |                        |          |             |               |  |      |      |       |      |      |        |                                                                    |
+| M001      | string    | uuid                   |    x     |      x      |               |  |  cw  |  x   |   x   |  x   |      |        |                                                                    |
+| M600      | timestamp | opprettetDato          |    x     |      x      |               |  |  cw  |  x   |   x   |  x   |      |        |                                                                    |
+| M601      | string    | opprettetAv            |    x     |      x      |               |  |  cw  |  x   |   x   |  x   |      |        |                                                                    |
+| M601      | string    | opprettetAvBrukerIdent |    x     |      x      |               |  |  cw  |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | adresseKnr             |    x     |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | postnr                 |          |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | poststed               |          |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | adressenavn            |    x     |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | nummer                 |          |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | adressekode            |          |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           | string    | alternativAdresse      |          |             |               |  |  x   |  x   |   x   |  x   |      |        |                                                                    |
+|           |           |                        |          |             |               |  |      |      |       |      |      |        |                                                                    |
+|           | ref       | refMappe               |    c     |             |               |  |      |  x   |   x   |      |  x   |        | <p>&bull; references **AbstraktMappe**</p><p>&bull; exactly one of **refMappe** and **refRegistrering** is required</p>        |
```

Comment:
Table formatting is a bit off in all national identifier tables.

Reply:
Addressed with [this commit](https://github.com/documaster/noark5-web-services/pull/37/commits/1d250c1a3df5b0886e8541db740d116c0064771b).